class-tgm-plugin-activation.php

```diff
@@ -2713,7 +2713,7 @@ protected function get_row_actions( $item ) {
 			}
 
 			$prefix = ( defined( 'WP_NETWORK_ADMIN' ) && WP_NETWORK_ADMIN ) ? 'network_admin_' : '';
-			return apply_filters( "tgmpa_{$prefix}plugin_action_links", array_filter( $action_links ), $item['slug'], $item, $this->view_context );
+			return apply_filters( 'tgmpa_' . $prefix . 'plugin_action_links', array_filter( $action_links ), $item['slug'], $item, $this->view_context );
```

Comment:
While we're not, there may be plugins and themes that are using it, so TGMPA shouldn't be the cause of them getting duff auto-generated documentation because it tripped up in some way.

Of course, we could use WP-Parser if we wanted to create some technical API documentation.